cfgov_setup.py

```diff
@@ -0,0 +1,66 @@
+import os
+
+from distutils.command.build_ext import build_ext as _build_ext
+from setuptools.command.bdist_egg import bdist_egg as _bdist_egg
+from wheel.bdist_wheel import bdist_wheel as _bdist_wheel
+
+from setuptools import Command
+
+from subprocess import call
+
+
+class FrontEndBuildFailure(Exception):
+    pass
+
+
+class build_frontend(Command):
+    """ A command class to run `setup.sh` """
+    description = 'build front-end JavaScript and CSS'
+    user_options = []
+    possible_scripts = ['frontendbuild.sh', 'setup.sh']
```

Comment:
In fact, we can go the other way, and not have a default, and do away with do_frontend_build too. frontend_build_script becomes the flag whose presence triggers the cfgov-setup magic,